treeherder/log_parser/tasks.py

```diff
@@ -49,64 +47,49 @@ def parse_job_log(func_name, routing_key, job_log):
         "parse_log": parse_log,
     }
 
-    task_priorities = {
-        "normal": 0,
-        "failures": 1,
-    }
-
-    callback_priority = "normal"
-
-    callback_group = []
-
     logger.debug("parse_job_log for job log %s (%s, %s)",
                  job_log.id, func_name, routing_key)
     priority = routing_key.rsplit(".", 1)[1]
-    if task_priorities[priority] > task_priorities[callback_priority]:
-        callback_priority = priority
 
     signature = task_funcs[func_name].si(job_log.id, priority)
     signature.set(routing_key=routing_key)
 
-    if func_name in ["parse_log", "store_failure_lines"]:
-        callback_group.append(signature)
-    else:
-        signature.apply_async()
-
-    if callback_group:
-        callback = crossreference_error_lines.si(job_log.job.id)
-        callback.set(routing_key="crossreference_error_lines.%s" %
-                     callback_priority)
-        create_taskset(callback_group, callback)
+    signature.apply_async()
 
 
 @retryable_task(name='log-parser', max_retries=10)
-@taskset
 @parser_task
-def parse_log(job_log, _priority):
+def parse_log(job_log, priority):
     """
     Call ArtifactBuilderCollection on the given job.
     """
     post_log_artifacts(job_log)
+    crossreference_error_lines.apply_async(
+        args=[job_log.job.id, priority],
+        routing_key="crossreference_error_lines.%s" % priority)
 
 
 @retryable_task(name='store-failure-lines', max_retries=10)
-@taskset
 @parser_task
 def store_failure_lines(job_log, priority):
     """Store the failure lines from a log corresponding to the structured
     errorsummary file."""
     logger.debug('Running store_failure_lines for job %s' % job_log.job.id)
     failureline.store_failure_lines(job_log)
-    if settings.AUTOCLASSIFY_JOBS:
-        autoclassify.apply_async(args=[job_log.job.id],
-                                 routing_key="autoclassify.%s" % priority)
+    crossreference_error_lines.apply_async(
+        args=[job_log.job.id, priority],
+        routing_key="crossreference_error_lines.%s" % priority)
 
 
 @retryable_task(name='crossreference-error-lines', max_retries=10)
-def crossreference_error_lines(job_id):
+def crossreference_error_lines(job_id, priority):
```

Comment:
If these are going to land together as part of the "stopping ingestion" plan, then that sounds fine.